brave/browser-laptop
# Disable autoplay

Test plan
---
A.
1. Go to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_video_autoplay
2. Video should not autoplay
3. Click bravey panel to allow autoplay
4. Video should autoplay

B.
1. Go to youtube and watch a video
2. Video should not autoplay
3. Click bravey panel to allow autoplay
4. Video should autoplay

C. Also covered by automatic test

---


Test Plan:

Resolves #7663
Resolves #2227 
Resolves #4045
